autoscale/src/test/java/com/sequenceiq/periscope/converter/DistroXAutoscaleScalingActivityResponseConverterTest.java

```diff
@@ -0,0 +1,70 @@
+package com.sequenceiq.periscope.converter;
+
+import static java.time.temporal.ChronoUnit.MINUTES;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Instant;
+import java.util.Date;
+
+import javax.inject.Inject;
+
+import org.junit.Test;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.StackType;
+import com.sequenceiq.periscope.api.model.ActivityStatus;
+import com.sequenceiq.periscope.api.model.ClusterState;
+import com.sequenceiq.periscope.api.model.DistroXAutoscaleScalingActivityResponse;
+import com.sequenceiq.periscope.domain.Cluster;
+import com.sequenceiq.periscope.domain.ClusterPertain;
+import com.sequenceiq.periscope.domain.ScalingActivity;
+
+public class DistroXAutoscaleScalingActivityResponseConverterTest {
+
+    private static final String TEST_ACTIVITY_CRN = "crn:cdp:autoscale:us-west-1:9d74eee4-1cad-45d7-b645-7ccf9edbb73d:datahubAutoscaleActivity:6d" +
+            "d833d4-7bc6-4a65-992a-b5da3400312c";
+
+    private static final String TEST_REASON = "test trigger reason";
+
+    private static final String CLOUDBREAK_STACK_CRN_1 = "crn:cdp:datahub:us-west-1:tenant:cluster:878605d9-f9e9-44c6-9da6-e4bce9570ef5";
+
+    private static final String TEST_TENANT = "testTenant";
+
+    @Inject
+    private DistroXAutoscaleScalingActivityResponseConverter underTest = new DistroXAutoscaleScalingActivityResponseConverter();
```

Comment:
Please remove the initialization. `@InjectMocks` will do it for you

Reply:
Changed

Reply:
Still hasn't changed.